torch/_export/serde/serialize.py

```diff
@@ -194,7 +207,47 @@ def deserialize_metadata(metadata) -> Dict[str, str]:
     module_fqn = metadata.get("module_fqn")
     if module_fqn is not None:
         ret["module_fqn"] = module_fqn
-    # TODO(angelayi) add nn_module_stack and source_fn
+
+    def deserialize_meta_func(serialized_target: str):
+        module = None
+        if serialized_target.startswith("torch.nn"):
+            module = torch.nn
+            serialized_target_names = serialized_target.split(".")[2:]
+        elif serialized_target.startswith("torch"):
+            module = torch
+            serialized_target_names = serialized_target.split(".")[1:]
+        else:
+            return deserialize_operator(serialized_target)
+
+        target = module
+        for name in serialized_target_names:
+            if not hasattr(target, name):
+                return serialized_target
+            else:
+                target = getattr(target, name)
+        return target
+
+    if nn_module_stack_str := metadata.get("nn_module_stack"):
```

Comment:
what's the difference between nn_module_stack and module_fqn?

Reply:
module_fqn seems like a sigmoid specific thing: https://fburl.com/code/uafys81f

Reply:
Mmm, worth standardizing these extra fields right now, maybe it's dupe / derivable from something else?

Reply:
Looking at the line, it looks like `module_fqn` is derivable from the stack. Not clear whether this is the "current" (innermost) module or the "main" (outermost) module?